flink-tests/src/test/java/org/apache/flink/test/streaming/runtime/DataStreamPojoITCase.java

```diff
@@ -139,17 +139,6 @@ public void flatMap(Data value, Collector<Data> out) throws Exception {
 	}
 
 
-	/**
-	 * As per FLINK-3702 Flink doesn't support nested pojo fields for sum()
-	 */
-	@Test(expected = IllegalArgumentException.class)
```

Comment:
I think it would be useful to keep this and `testNestedPojoFieldAccessor`. So that we also have an end-to-end test.